src/i18n/i18n.js

```diff
@@ -0,0 +1,69 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+const translationsEn = {
+  inputPlaceholder: 'Movies, person...',
+  pop: 'Popular',
+  top: 'Top rated',
+  up: 'Upcoming',
```

Comment:
it's hard to read, seeing this keys in components

Reply:
Fixed!